recipes/example/meta.yaml

```diff
@@ -1,10 +1,10 @@
 # Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
 
 # Jinja variables help maintain the recipe as you'll update the version only here.
-# Using the name variable with the URL in line 14 is convenient
-# when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
+  # Using the name variable with the URL in line 14 is convenient
+  # when copying and pasting from another recipe, but not really needed.
```

Comment:
could you please rever the example?

Reply:
No problem. My IDE was adding auto formatting on commit and adding spaces. Being a bit of a novice I am having trouble just getting git to revert, but the example file should be identical to the original now.